src/main/java/uk/gov/hmcts/reform/sendletter/exception/LetterNotFoundException.java

```diff
@@ -1,11 +1,18 @@
 package uk.gov.hmcts.reform.sendletter.exception;
 
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+import uk.gov.hmcts.reform.logging.exception.UnknownErrorCodeException;
+
 import java.util.UUID;
 
-public class LetterNotFoundException extends RuntimeException {
+/**
+ * SonarQube reports as error. Max allowed - 5 parents
+ */
+@SuppressWarnings("squid:MaximumInheritanceDepth")
+public class LetterNotFoundException extends UnknownErrorCodeException {
 
     public LetterNotFoundException(String letterId, Throwable cause) {
-        super("Letter with ID '" + letterId + "' not found", cause);
+        super(AlertLevel.P4, "Letter with ID '" + letterId + "' not found", cause);
```

Comment:
Why is this P4 compared to P1 for the other exceptions?

Reply:
`LetterNotFound` is kind of customer error and it is the least priority support should care about hence P4